pom.xml

```diff
@@ -228,6 +228,13 @@
             <artifactId>velocity-tools</artifactId>
             <version>1.4</version>
         </dependency>
+
+        <dependency>
+            <groupId>oro</groupId>
+            <artifactId>oro</artifactId>
+            <version>2.0.8</version>
+        </dependency>
```

Comment:
I was not able to run this code because of
``
ERROR StatusLogger No log4j2 configuration file found. Using default configuration: logging only errors to the console.
Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/oro/text/perl/MalformedPerl5PatternException
``